### DISCUSSION TITLE
A question about download files

### DISCUSSION THREAD

**Discussion by CNwangbin** [geneontology/helpdesk#398] [Q&A]
First, wether the obo file equal to owl file?
Second, in obo file,there are some terms with "intersection of" tags.
I don't konw this meaning.
Could I see them as the equalent definition of GO:0000022?
![image](https://user-images.githubusercontent.com/87804802/172115459-ea31324f-51a7-4746-afc7-1eced851041c.png)

**Comment by balhoff** [comment]
@CNwangbin the OBO file is intended to be equivalent to the OWL file, especially for the main release files at `http://purl.obolibrary.org/obo/go.obo` and `http://purl.obolibrary.org/obo/go.owl`. There are some features of OWL that are not representable in OBO files, but these features aren't generally found in the main GO file. You're correct, the `intersection_of` tags define an OWL equivalentClass expression. You can find a translation from OBO to OWL axioms here: https://owlcollab.github.io/oboformat/doc/obo-syntax.html
